tdrs-backend/tdpservice/stts/tests.py

```diff
@@ -0,0 +1,5 @@
+"""STTs Unit Testing."""
+
+# from django.test import TestCase
+
+# Create your tests here.
```

Comment:
I noticed that this test class is empty, do we need to add unit tests here? Why or why not? Thanks!